test/unit/spec/util/BootstrapDialogSpec.js

```diff
@@ -98,10 +102,11 @@ define(['jquery', 'util/bootstrapDialog'], ($, Dialog) => {
             simpleDialog = new Dialog({
                 title: 'Test for command on enter keypress',
                 body: $('<div>'),
-                buttons: [$('<button>').append('Clickable button').click(btnFn)],
+                buttons: [
```

Comment:
add in the `data-dismiss` to ensure that the modal gets eliminated -- otherwise it hangs around in the DOM